### DISCUSSION TITLE
Hide the Upgrade button for accounts that have already upgraded

### DISCUSSION THREAD

**Discussion by numbermess** [Kong/insomnia#6578] [Feedback]
I am using the paid version of Insomnia only for myself and have no plans to use it with a team. The new "Upgrade" button that is always present in the top right corner of version 8 is distracting and unnecessary clutter and I don't want to see it. 

I couldn't find a way to disable it in the settings, so hopefully there will be a way to do that soon.

**Comment by benoitlemaire** [comment]
Totally agree with @numbermess. The first time I saw it, I thought my plan had been downgraded to basic, but I'm already on the Pro plan

**Comment by subnetmarco** [comment]
This button has since then been removed and hidden under the user dropdown.
